tests/logger/test_log_collector.py

```diff
@@ -584,33 +539,28 @@ def test_enter_leave_block_section_all(
 
     def test_get_faults(
         self,
-        _faults_df,
+        faults_log_collector_df,
         logger,
-        _platform_blocked_fault_configuration: PlatformBlockedFaultConfiguration,
-        _track_blocked_fault_configuration: TrackBlockedFaultConfiguration,
-        _track_speed_limit_fault_configuration: TrackSpeedLimitFaultConfiguration,
-        _schedule_blocked_fault_configuration: ScheduleBlockedFaultConfiguration,
-        _train_prio_fault_configuration: TrainPrioFaultConfiguration,
-        _train_speed_fault_configuration: TrainSpeedFaultConfiguration,
+        platform_blocked_fault_configuration,
+        track_blocked_fault_configuration,
+        track_speed_limit_fault_configuration,
+        schedule_blocked_fault_configuration,
+        train_prio_fault_configuration,
+        train_speed_fault_configuration,
         log_collector: LogCollector,
     ):
+        # pylint: disable=R0801
```

Comment:
What do you disable here? Maybe you could use the name instead of this "identifier"

Reply:
It supresses a duplicate code warning, because the setup faults call is 9 lines long, changed it to the identifier name